Api/Endpoints/Endpoint.php

```diff
@@ -27,9 +26,9 @@ class Endpoint implements EndpointInterface
      * Endpoint constructor.
      *
      * @param string $baseUrl parent::constant
-     * @param string $apiToken Client provided authentication token
+     * @param string|null $apiToken Client provided authentication token
      */
-    public function __construct($baseUrl, $apiToken)
+    public function __construct(string $baseUrl, ?string $apiToken)
```

Comment:
Why is API token suddenly optional?

Reply:
@emsyzz Good question. Actually, it was nullable before (based on annotations). Fixed. `$apiToken` is not nullable anymore.